### PR TITLE
Updates "Meeting: What to expect"

### PR DESCRIPTION
**Pull request by Crixu**
I updates the part about "Meeting: What to expect" with a generic overview of the agenda of the team.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -68,10 +68,22 @@ You can [sign up](https://wordpress.slack.com/signup) using your WordPress.org u
...
+
+The meeting is usually to connect about WordPress happenings throughout the week connected with hosting and to catch up on the status of the team's ongoing projects.
+The agenda mainly consists of the following topics:
+
+* Greetings
+* Highlights
+* Hosting Team tasks
+* Open Floor
+
+During the Greetings we do a “Wave in” to get an overview who is in for the meeting and check how everyone is doing. New attendees are welcomed to introduce themselves.
```
Comment (getsource) [inline readme.md:80]: Do you think we should include the standard greeting -- or that should be on a different page?

File: readme.md [hunk 2]
```diff
@@ -68,10 +68,22 @@ You can [sign up](https://wordpress.slack.com/signup) using your WordPress.org u
...
+
+In the highlights section the team talks about the latest happenings in the WordPress space and it's impacts on the hosts. Examples are new WordPress releases, new proposals or WordCamps.
+
+The Hosting team tasks part of the meeting focuses on open tasks or issues on hosting related projects like the handbook or the PHPUNit tests. Additionally, we handle request from other teams here.
```
Comment (getsource) [inline readme.md:84]: Maybe `automated tests` or "hosting tests" instead of `PHPUNit tests`?

File: readme.md [hunk 3]
```diff
@@ -68,10 +68,22 @@ You can [sign up](https://wordpress.slack.com/signup) using your WordPress.org u
...
+The agenda mainly consists of the following topics:
+
+* Greetings
+* Highlights
+* Hosting Team tasks
+* Open Floor
+
+During the Greetings we do a “Wave in” to get an overview who is in for the meeting and check how everyone is doing. New attendees are welcomed to introduce themselves.
+
+In the highlights section the team talks about the latest happenings in the WordPress space and it's impacts on the hosts. Examples are new WordPress releases, new proposals or WordCamps.
```
Comment (getsource) [inline readme.md:82]: "their impacts on hosts"

File: readme.md (hunk 1 above)
Comment (Crixu) [reply readme.md:80]: Maybe we can have another page like a "Standard Meeting" Page where we keep track of all our standard greetings

Comment (getsource) [reply readme.md:80]: Yeah, I like that. Maybe "Running a Meeting" or something?

**Pull request merged**